src/LogViewer/chunk-worker.js

```diff
@@ -127,6 +143,18 @@ const getParagraphClass = (lineNumber, { highlightStart, highlightEnd }) => {
     '';
 };
 
+// Tagged template function
+function escapeHtml(pieces) {
```

Comment:
We don't use function declarations in this repo, so let's switch to expressions.